tests/layer_tests/pytorch_tests/test_shape_as_tensor.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import torch
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+class aten_shape_as_tensor(torch.nn.Module):
+    def __init__(self) -> None:
+        torch.nn.Module.__init__(self)
+
+    def forward(self, input_tensor):
+        return torch.ops.aten._shape_as_tensor(input_tensor)
+
+class TestShapeAsTensor(PytorchLayerTest):
+    def _prepare_input(self):
+        return (self.input_tensor,)
+
+    @pytest.mark.parametrize("shape", [
+        (1,2,3,4), 
```

Comment:
please test scalar case

Reply:
Thank you for this suggestion, apparently returning a 0-dim array as a shape is not allowed in OV as mentioned below. I commented out the test and will re-enable it when fix is available.